ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/vlan-interfaces.j2

```diff
@@ -1,156 +1,166 @@
-{%         if vlan_interfaces[vlan_interface].ip_address_virtual is defined %}
+{%     endif %}
+{%     if vlan_interfaces[vlan_interface].ip_address_virtual is arista.avd.defined %}
    ip address virtual {{ vlan_interfaces[vlan_interface].ip_address_virtual }}
-{%         endif %}
-{%         if vlan_interfaces[vlan_interface].ip_helpers is defined and vlan_interfaces[vlan_interface].ip_helpers is not none %}
-{%            for ip_helper in vlan_interfaces[vlan_interface].ip_helpers | arista.avd.natural_sort %}
-   ip helper-address {{ ip_helper }} {% if vlan_interfaces[vlan_interface].ip_helpers[ip_helper].vrf is defined and vlan_interfaces[vlan_interface].ip_helpers[ip_helper].vrf is not none %}vrf {{vlan_interfaces[vlan_interface].ip_helpers[ip_helper].vrf }} {% endif %} {% if vlan_interfaces[vlan_interface].ip_helpers[ip_helper].source_interface is defined and vlan_interfaces[vlan_interface].ip_helpers[ip_helper].source_interface is not none %}source-interface {{vlan_interfaces[vlan_interface].ip_helpers[ip_helper].source_interface }}{% endif %}
-
-{%            endfor %}
-{%         endif %}
-{%         if vlan_interfaces[vlan_interface].ipv6_enable is defined and vlan_interfaces[vlan_interface].ipv6_enable == true %}
+{%     endif %}
+{%    for ip_helper in vlan_interfaces[vlan_interface].ip_helpers | arista.avd.natural_sort %}
+{%        set ip_helper_cli = "ip helper-address " ~ ip_helper %}
+{%        if vlan_interfaces[vlan_interface].ip_helpers[ip_helper].vrf is arista.avd.defined %}
+{%           set ip_helper_cli = ip_helper_cli ~ " vrf " ~ vlan_interfaces[vlan_interface].ip_helpers[ip_helper].vrf %}
+{%        endif %}
+{%        if vlan_interfaces[vlan_interface].ip_helpers[ip_helper].source_interface is arista.avd.defined %}
+{%            set ip_helper_cli = ip_helper_cli ~ " source-interface " ~ vlan_interfaces[vlan_interface].ip_helpers[ip_helper].source_interface %}
+{%        endif %}
+   {{ ip_helper_cli }}
+{%     endfor %}
+{%     if vlan_interfaces[vlan_interface].ipv6_enable is arista.avd.defined(true) %}
    ipv6 enable
-{%         endif %}
-{%         if vlan_interfaces[vlan_interface].ipv6_address is defined and vlan_interfaces[vlan_interface].ipv6_address is not none %}
+{%     endif %}
+{%     if vlan_interfaces[vlan_interface].ipv6_address is arista.avd.defined %}
    ipv6 address {{ vlan_interfaces[vlan_interface].ipv6_address }}
-{%         endif %}
-{%         if vlan_interfaces[vlan_interface].ipv6_address_link_local is defined and vlan_interfaces[vlan_interface].ipv6_address_link_local is not none %}
+{%     endif %}
+{%     if vlan_interfaces[vlan_interface].ipv6_address_link_local is arista.avd.defined %}
    ipv6 address {{ vlan_interfaces[vlan_interface].ipv6_address_link_local }} link-local
-{%             endif %}
-{%         if vlan_interfaces[vlan_interface].ipv6_nd_ra_disabled is defined and vlan_interfaces[vlan_interface].ipv6_nd_ra_disabled == true %}
-   ipv6 nd ra disabled
 {%         endif %}
```

Comment:
fix indentation

Reply:
Fixed with  07d5adb6

Reply:
not seeing the change here. Line 60 is still indented much more than line 58.